workalendar/america/barbados.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, division, print_function
+from datetime import date
+from dateutil import easter
+
+from ..core import WesternCalendar, ChristianMixin
+from ..core import Calendar, SUN, MON
+from ..registry import iso_register
+
+
+@iso_register("BB")
+class Barbados(ChristianMixin, WesternCalendar):
+    "Barbados"
+
+    include_good_friday = True
+    include_easter_sunday = True
+    include_easter_monday = True
+    include_whit_monday = True
+    include_boxing_day = True
+    shift_sunday_holidays = True
+    EASTER_METHOD = easter.EASTER_WESTERN
+
+    # All holiday are shifted if on a Sunday
+    almost_fixed_holidays = WesternCalendar.FIXED_HOLIDAYS + (
```

Comment:
when days are shifted to the next monday, we usually keep them in the calendar, but add a day with the "label + shift", for example:

* "Day of the Tentacle" happens on SUN,
* We add a "Day of the Tentacle (shifted)" on the next MON.

wouldn't this work here? instead of *replacing* the days in the ``get_variable_days`` method?

Reply:
I've changed the behavior sot that it's consistent with the other calendars.

Reply:
I've had an after thought about this. I think I'll make a PR on your PR, wait and see, I think it's more elegant to get rid of the ``almost_fixed_holidays`` and do the "sunday" shift in the ``get_fixed_holidays``.
Stay tuned.